sdk/examples/how_tos/native_tokens/destroy_foundry.rs

```diff
@@ -12,7 +12,11 @@
 //! cargo run --release --all-features --example destroy_foundry
 //! ```
 
-use iota_sdk::{wallet::Result, Wallet};
```

Comment:
Can't comment above but `This is only possible if its circulating supply is 0` in the doc seems wrong